### ISSUE TITLE
Node Tree does not always execute from the Video Sequence Editor

### ISSUE THREAD

**Issue by JaredOwenAnimations** [JacquesLucke/animation_nodes#940]
[AnimationNodes_VSE_Bug.zip](https://github.com/JacquesLucke/animation_nodes/files/2353337/AnimationNodes_VSE_Bug.zip)
I've attached the project that I'm describing.

I have a main scene with some animated custom properties and an empty.
Using Animation Nodes I have 
- two separate cubes that either move up and down or squish when the properties change.
- another cube that changes scale according to the z value of the empty
- a text object that displays the current frame

Then I create another scene, go the VSE, and add the main scene as a scene strip.
The node tree doesn't always execute as it should (or execute at all).

With this small "toy" project I can usually get it to work by either deleting the scene strip and adding it again OR restarting the blender.  However, I have some larger projects where this does not work.  Everything I'm doing can be done with drivers but I really really like Animation Nodes and I would like to integrate this into my regular work flow.

Is this a known bug?  Is AN meant to work with scene strips in the VSE?

**Comment by OmarSquircleArt** [comment]
It seems to be working for me. What version are you using?

**Comment by JaredOwenAnimations** [comment]
Thanks for your quick response!

Here are the versions I'm using:
	Blender 2.79b
	Animation Nodes 2.0.4

[AN_VSE_Bug_MultipleScenes.zip](https://github.com/JacquesLucke/animation_nodes/files/2353784/AN_VSE_Bug_MultipleScenes.zip)

I think I found a better way to reproduce.  The problem appears when there are multiple scenes.  It seems as though the node tree only wants to execute for the very first scene.  In this new example I have 3 scenes:
"CubeScene" - has a cube and a node tree to move the X location
"SphereScene" - has a sphere and a node tree to move the Y location
"VideoEditing" - VSE with both scene strips

(I included a monkey spining so that I could tell it was displaying regular animation correctly)

As is - the cube seems to be working correctly but the sphere doesn't move.

Now go and delete the "CubeScene".  Go back to the VideoScene - delete the sphere scene strip and re-add it.  Now the sphere scene works!

Does AN only execute for the first scene in the scene list?
